### ISSUE TITLE
Fixing PSR2 multiline nested call is confused by multiline array

### ISSUE THREAD

**Issue by VasekPurchart** [squizlabs/PHP_CodeSniffer#578]
When fixing a nested multiline call, the indentation of the closing parentheses are indented to the level of the wrongly indented parenthesis (which should be fixed). It does not work in this special combination although it works in very similar scenarios as shown below.

**Error case**:

Code:
```php
<?php

foo(bar(1,
	[
		'foo'
	])
);
```
Fixed code:
```php
<?php

foo(bar(1,
	[
		'foo'
	]
	));
```
Expected code:
```php
<?php

foo(bar(1,
	[
		'foo'
	]
)); // <--
```
---

**Works fine when not nested**: (fine except separate bug reported in #577)

Code:
```php
<?php

bar(1,
	[
		'foo'
	]);
```
Fixed code:
```php
<?php

bar(1,
	[
		'foo'
	]
);
```
---

**Works fine when not array is not multiline**: (fine except separate bug reported in #577)

Code:
```php
<?php

foo(bar(1,
	['foo'])
);
```
Fixed code:
```php
<?php

foo(bar(1,
	['foo']
));
```
---

Everything tested on 2.3.1.

**Comment by gsherwood** [comment]
When I run PHPCBF over your code, I get this:
```php
<?php

foo(bar(
    1,
    [
        'foo'
    ]
));
```

Are you using the latest version of PHPCS?

**Comment by VasekPurchart** [comment]
This is the same problem as here https://github.com/squizlabs/PHP_CodeSniffer/issues/577#issuecomment-97405768 . When I delete the exclude it works as expected. But here I see no connection with the `ContentAfterOpenBracket`.

Are the fixes intended to work correctly even if some parts are excluded like this? I've quite a lot of other excludes, because I try to use as much as possible from existing sniffs before writing my own, but that means I differ in the details.

**Comment by gsherwood** [comment]
See #577. Closing this one.
